py_zipkin/zipkin.py

```diff
@@ -143,6 +154,17 @@ def __init__(
         if self.sample_rate is not None and not (0.0 <= self.sample_rate <= 100.0):
             raise ZipkinError('Sample rate must be between 0.0 and 100.0')
 
+        if not set(include).issubset(set(STANDARD_ANNOTATIONS.keys())):
```

Comment:
Sorry for the late comment. It'd be nice if the `set(STANDARD_ANNOTATIONS.keys())` happened out of the context of this function. Something like `ANNOTATION_KEYS = fronzenset(STANDARD_ANNOTATIONS.keys())` in the global scope. Sound reasonable? The less work we can do at runtime the better, IMO.